modules/ROOT/pages/osv2-apis.adoc

```diff
@@ -66,158 +125,332 @@ The following table compares the Object Store REST API operations to Object Stor
+the object store.
+. Send the operation to the Object Store Stats API.
+
+
+=== Examples: Object Store Stats API
+
+* <<Example: Retrieve Usage Metrics by Organization>>
+* <<Example: Retrieve Usage Metrics by Environment>>
+
+==== Example: Retrieve Usage Metrics by Organization
+
+To view Object Store usage statistics across the master business organization
+between July 11, 2020 and September 11, 2020,
+submit a `curl` command like the following, 
+replacing ORG_ID with your organization ID and BEARER_TOKEN with your bearer token:
+
+
+[source,console,linenums]
+----
+curl -X GET \
+  'https://object-store-stats.anypoint.mulesoft.com/api/v1/organizations/ORG_ID?startDate=2020-07-11T17%3A51%3A54.000Z&endDate=2020-09-11T17%3A51%3A54.000Z&period=1month' \
+  -H 'authorization: Bearer BEARER_TOKEN'
+----
+
+This request returns a response like the following:
+
+[source,json,linenums]
+----
+[
+ {"timeStamp":"2020-07-01T00:00:00Z","objectStoreRequestCount":0},
+ {"timeStamp":"2020-08-01T00:00:00Z","objectStoreRequestCount":38},
+ {"timeStamp":"2020-09-01T00:00:00Z","objectStoreRequestCount":28}
+]
+----
+
+
+
+==== Example: Retrieve Usage Metrics by Environment 
+
+To view Object Store usage statistics for an environment between June 13, 2020 and September 13, 2020,
+submit a `curl` command like the following, 
+replacing ORG_ID with your organization ID, ENV_ID with your environment ID,
+and BEARER_TOKEN with your bearer token:
+
 
 [source,console,linenums]
 ----
 curl -X GET \
-> -H 'authorization: bearer BEARER_ID' \
-> 'https://object-store-us-east-1.anypoint.mulesoft.com/api/v1/organizations/ORG_ID/environments/ENV_ID/stores'
+  'https://object-store-stats.anypoint.mulesoft.com/api/v1/organizations/ORG_ID/environments/ENV_ID?startDate=2020-06-13T17%3A51%3A54.000Z&endDate=2020-09-13T17%3A51%3A54.000Z&period=1month' \
+  -H 'authorization: Bearer BEARER_TOKEN'
 ----
 
-The output has values similar to:
+This request returns a response like the following:
 
 [source,json,linenums]
 ----
-{"values":[{"storeId":"APP_demo-aug12_DEFAULT_USER_STORE","encrypted":true,"permanentOsFlag":false,
-  "persistent":true,"defaultTtlSeconds":1209600,"defaultConfirmationTtlSeconds":600},
-{"storeId":"APP_objectstore42_DEFAULT_USER_STORE","encrypted":false,"permanentOsFlag":false,
-  "persistent":true,"defaultTtlSeconds":1209600,"defaultConfirmationTtlSeconds":600},
-{"storeId":"APP_objectstore42_mypartition","encrypted":false,"permanentOsFlag":false,
-  "persistent":true,"defaultTtlSeconds":1209600,"defaultConfirmationTtlSeconds":600}],"nextPageToken":null}
+[
+ {"timeStamp":"2020-06-01T00:00:00Z","objectStoreRequestCount":3},
+ {"timeStamp":"2020-07-01T00:00:00Z","objectStoreRequestCount":30},
+ {"timeStamp":"2020-08-01T00:00:00Z","objectStoreRequestCount":1},
+ {"timeStamp":"2020-09-01T00:00:00Z","objectStoreRequestCount":25}
+]
 ----
 
+
+== Download Object Store APIs from Exchange 
```

Comment:
Shouldn't this come at the top of the set rather than the bottom? A prereq?

Reply:
No - you don't need to download to use.